src/browser/appwindow.js

```diff
@@ -99,8 +109,11 @@ AppWindow.prototype.initYoProcess = function () {
   this.yoProcess.on('message', function (msg) {
     console.log('APP', msg);
 
-    this.sendCommandToBrowserWindow(msg.event, msg.data);
+    if (msg.event === 'generator:done') {
+      this.openProject(msg.data);
+    }
```

Comment:
I have the feeling we will have more and more conditions like this. Therefore I would prefer to emit all incoming events to the `AppWindow` instance and add listeners for all events you are interested in.